README.md

```diff
@@ -1,36 +1,300 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15575410&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
 
+Software engineering is the process that involves the design, development, testing, and maintenance of software applications. It involves the application of engineering principles to create reliable, scalable, and robust software solutions that meet user needs.
+It’s important in the technology industry because: -
```

Comment:
Object-Oriented Programming- the introduction of OOP paradigms provided a more modular and reusable approach to building softwarwe systems.
Agile software development - introduced a new approach to software development that emphasized flexibility and adaptability.
The Software Crisis (Late 1960s): This period marked a realization of the challenges associated with large-scale software development. Projects were often over budget, behind schedule, and delivered with poor quality.